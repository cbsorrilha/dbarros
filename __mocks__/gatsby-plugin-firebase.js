module.exports = {
    FirebaseContext: {
        _context: {
            Consumer: null,
            Provider: null,
        },
    },
}