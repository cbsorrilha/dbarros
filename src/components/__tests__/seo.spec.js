import React from "react"
import renderer from "react-test-renderer"
import SEO from '../seo';
import { useStaticQuery } from '../../../__mocks__/gatsby';

describe('Snapshot testing the SEO', () => {
    it("renders correctly", () => {
        const props = {
            description: 'description',
            meta: [],
            title: 'title'
        }
        useStaticQuery.mockReturnValue({
            site:{
                siteMetadata: {
                    title: 'title',
                    description: 'description',
                    author: 'author',
                }
            }
        });
        const tree = renderer.create(<SEO {...props} />).toJSON()
        expect(tree).toMatchSnapshot()
    })
})
