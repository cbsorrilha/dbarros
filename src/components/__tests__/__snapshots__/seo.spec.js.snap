// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Snapshot testing the SEO renders correctly 1`] = `null`;
